pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
```

Comment:
The constraints have to include a list of values. Example use case - a volume that can be accessed from multiple nodes or racks.

```go
TopologyConstraints []TopologyConstraint

type TopologyConstrating struct {
    key string
    value []string // `or` is the assumed operator
}
```